core/graphelier-service/api/hndlrs/msghandler.go

```diff
@@ -29,7 +29,7 @@ func FetchMessages(env *Env, w http.ResponseWriter, r *http.Request) error {
 		return StatusError{400, err}
 	}
 	paginator := models.Paginator{NMessages: intNMessages, SodOffset: intSodOffset}
-	messages, err := env.Connector.GetMessagesWithPagination(instrument, intSodOffset, &paginator)
+	messages, err := env.Connector.GetMessagesWithPagination(instrument, &paginator)
```

Comment:
im guessing passing `intSodOffset` was useless because the paginator object already contained it ?

Reply:
Yes sir